backend/app/models/quiz.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, TYPE_CHECKING, Optional
+
+from sqlalchemy import func, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.models.base import Base
+
+if TYPE_CHECKING:
+    from app.models.user import User
+    from app.models.session import Session
+
+
+class Quiz(Base):
+    __tablename__ = "quiz"
+
+    id: Mapped[str] = mapped_column(primary_key=True, server_default=func.gen_random_uuid())
+    name: Mapped[str]
```

Comment:
I'm still quite new to FastAPI and SQLAlchemy so I have not yet formed a firm opinion - Should we validate the user import (e. g. string length) in both ORM models and the FastAPI schema?

Reply:
I'd go for as little constaint on DB level as possible. I think pydantic validation already does the trick and any modifications of DB data will probably origin for pydantic models.